### ISSUE TITLE
Garbage on touch due to Marshaller

### ISSUE THREAD

**Issue by badlogic** [robovm/robovm#311]
Hi,

i've been investigating the GC invocations that happen in libgdx games that rely on touch events. General reproduction scenario:
- Create an empty libgdx app
- Launch on simulator/device, attach Instruments
- Observe memory usage while not touching and moving the finger across the screen, no GCed resources are allocated
- Observe memory usage while touching and moving the finger across the screen, tons of objects are created and eventually GCed.

Here's what it looks like memory-wise when you move a finger across the screen:

![memory-usage](http://libgdx.badlogicgames.com/uploads/Screen%20Shot%202014-04-10%20at%2001.34.38-APmEFBURnu.png)

I first suspected an issue in libgdx's IOSInput class. To get touch events, we use a [GLKView in IOSGraphics] that responds to touch events. We override the touch related methods of that view and push the [NSSet<UITouch> to IOSInput](https://github.com/libgdx/libgdx/blob/master/backends/gdx-backend-robovm/src/com/badlogic/gdx/backends/iosrobovm/IOSInput.java#L496) which is responsible for processing and handing those touch events to the actual game code.

I naturally suspected our IOSInput#toTouchEvents method to be the thing that generates garbage. To test this assumption, i removed all the code in that method. The result was the same, something generates objects like crazy. Following the call stack upwards back to GLKView it is appearant that nothing on the libgdx is actually allocating anything. 

So the culprit must be the code that calls the GLKView#touch methods upon receiving touch events from the system, or the GLKView#touch methods themselves (which we call in the overrides). To make sure it's not the super#touch calls we do in our GLKView subclass, i simply removed the overrides and checked the effect with instruments. The results where still the same. I symbolicated the trace, here's the result:

![trace](http://libgdx.badlogicgames.com/uploads/Screen%20Shot%202014-04-10%20at%2001.52.34-QoR5xhje2K.png)

naturally, the touchesMoved methods of UIResponder is called most often, so it generates the most garbage. It appears that the UIResponder#touchXXX methods invoke the Marshaller which generates UITouch and NSSet objects.

Long story short: this is a problem for games that use touch as input, as it will lead to GC hiccups every few seconds. Is there any way to turn of marshalling and get at the raw ObjC touch event objects without generating GC?
